### DISCUSSION TITLE
File Open

### DISCUSSION THREAD

**Discussion by thistlillo** [mikekazakov/nimble-commander#431] [General]
In the "File" menu, there are two items for the command "Open": see screenshot  (I modified the shortcut for one of them to CMD+O).

<img width="285" alt="Screenshot 2024-10-17 at 00 32 00" src="https://github.com/user-attachments/assets/daefb0ee-0703-4711-accd-39595376084c">

The hotkeys pane of the preferences contains two items for the two menu items. There are no additional info here.

What's their difference?

**Comment by mikekazakov** [comment]
https://github.com/mikekazakov/nimble-commander/blob/main/Docs/Help.md#hotkeys
The first one enters the focused item, which has different meaning depending on the item type.
The second always open the item using default macOS associations.
